usr/log.h

```diff
@@ -97,4 +100,24 @@ do {									\
 } while (0)
 #endif
 
+#define LONGHORN_ERROR_LOG(fmt, ...) do { \
+    struct timeval tv; \
+    gettimeofday(&tv, NULL); \
+    struct tm tm_info; \
+    localtime_r(&tv.tv_sec, &tm_info); \
+    char buf[32]; \
+    strftime(buf, sizeof(buf), "%Y-%m-%d %H:%M:%S", &tm_info); \
+    eprintf("[%s.%06ld] " fmt "\n", buf, (long)tv.tv_usec, ##__VA_ARGS__); \
+} while (0)
+
+#define LONGHORN_DEBUG_LOG(fmt, ...) do { \
+    struct timeval tv; \
+    gettimeofday(&tv, NULL); \
+    struct tm tm_info; \
+    localtime_r(&tv.tv_sec, &tm_info); \
+    char buf[32]; \
+    strftime(buf, sizeof(buf), "%Y-%m-%d %H:%M:%S", &tm_info); \
+    dprintf("[%s.%06ld] " fmt "\n", buf, (long)tv.tv_usec, ##__VA_ARGS__); \
+} while (0)
```

Comment:
Both macros duplicate the same timestamp generation logic, Can we extract into an inline helper function?